pkg/pagerduty/pagerduty.go

```diff
@@ -62,7 +62,38 @@ func (p PagerDuty) MoveToEscalationPolicy(incident Incident, escalationPolicy Es
 	}
 	err := p.manageIncident(o)
 	if errors.Is(err, FailedToUpdateIncidentError{}) {
-		return pkgerrors.Wrap(err, "could not update the escalation policy")
+		return fmt.Errorf("could not update the escalation policy: %w", err)
+	}
+	return err
+}
+
+// AssignToCurrentUser will assign the incident to the current logged in user
+func (p PagerDuty) AssignToCurrentUser(incident Incident) error {
```

Comment:
not sure right now, but isnt the preferred way to write?:

`func (p *Pagerduty)`

Reply:
No, we use interfaces only if we modify the data.

I always use the non pointer of I can help it